resources/assets/tconstruct/models/block/tinker_tank_active.json

```diff
@@ -0,0 +1,35 @@
+{
+    "textures": {
+		"particle": "tconstruct:blocks/smeltery/tinker_tank",
+        "front": "tconstruct:blocks/smeltery/tinker_tank_overlay",
+        "side": "tconstruct:blocks/smeltery/seared_brick",
+        "back": "tconstruct:blocks/smeltery/tinker_tank_background"
+    },
+    "elements": [
+        {
+            "__comment": "main",
+            "from": [ 0, 0, 0 ], 
+            "to": [ 16, 16, 16 ], 
+            "faces": {
+                "north": { "texture": "#front", "uv": [ 0, 0, 16, 16 ], "cullface": "north" },
+                "east":  { "texture": "#side",  "uv": [ 0, 0, 16, 16 ], "cullface": "east"  },
+                "south": { "texture": "#side",  "uv": [ 0, 0, 16, 16 ], "cullface": "south" },
+                "west":  { "texture": "#side",  "uv": [ 0, 0, 16, 16 ], "cullface": "west"  },
+                "up":    { "texture": "#side",  "uv": [ 0, 0, 16, 16 ], "cullface": "up"    },
+                "down":  { "texture": "#side",  "uv": [ 0, 0, 16, 16 ], "cullface": "down"  }
+            }
+        },
+        {
+            "__comment": "backface",
+            "from": [ 0, 0, 1 ], 
+            "to": [ 16, 16, 0 ], 
+            "faces": {
+                "north": { "texture": "#back", "uv": [ 0, 0, 16, 16 ], "cullface": "north" },
```

Comment:
It's back about 1 pixek, so you can see a bit of the empty edge from the right angle